examples/text_labels_and_annotations/accented_text.py

```diff
@@ -1,21 +1,24 @@
 r"""
-matplotlib supports accented characters via TeX mathtext
+=================================
+Using accented text in matplotlib
+=================================
 
+Matplotlib supports accented characters via TeX mathtext.
```

Comment:
Also supported through regular Unicode characters, though that's not shown here.